lib/MusicBrainz/Server/Controller/Artist.pm

```diff
@@ -146,9 +146,12 @@ after 'load' => sub
         }
     }
 
-    $c->model('ArtistType')->load($artist, map { $_->target } @{ $artist->relationships_by_type('artist') });
-    $c->model('Gender')->load($artist);
-    $c->model('Area')->load($artist);
+    my $lang = $c->stash->{current_language} // 'en';
+    # TODO: This loads the artist's aliases to look for a primary alias in the
+    # user's language, but note that aliases are also being loaded by the show
+    # subroutine to get the artist's legal name.
```

Comment:
Is loading the aliases twice like this a concern, or should I just delete this TODO?

Reply:
It'd be nice if we could avoid it, since some artists have a ton of aliases. Perhaps add an `aliases` attribute to `Entity::Artist` to cache these on? Or store them in the stash somehow?

Reply:
Thanks, I've written some code to store these in `Entity::Artist` that seems to work.